Sources/BraintreeCore/BTGraphQLHTTP.m

```diff
@@ -63,12 +63,22 @@ - (void)handleRequestCompletion:(NSData *)data
 {
     // Network error
     if (error) {
-        [self callCompletionBlock:completionBlock body:nil response:nil error:error];
+        [self callCompletionBlock:completionBlock body:nil response:(NSHTTPURLResponse *)response error:error];
```

Comment:
We are including the response in other error cases but not here, this will bring our error completion here in line with our other error cases.